src/groundStation/uTransceiver.py

```diff
@@ -0,0 +1,130 @@
+import ctypes
+from ctypes import cdll, c_int, c_char
+import sys
+import zmq
+import threading
+import os.path
+import socket
+import time
+from enum import Enum
+import os
+
+# command format follows https://docs.google.com/spreadsheets/d/1zNhxhs0KJCp1187Vm3-zAzQHCY31f77l-0nlQmfXu1w/edit#gid=565953736
+# but with UHFDIR_ instead of UHF_ prefix. See document for examples
+
+#After sending a uhf-direct command, terminal displays all rx'd raw ASCII data for listentimeout_s
+
+class UHF_return(Enum):
```

Comment:
See if you can access this enum from the library or something. Or we can create python bindings